File: frontend/src/types/event.ts
export interface Schedule {
  id?: number;
  date: string;
  startTime: string;
  endTime: string;
  room: string;
  speaker: string;
}

export interface Event {
  id?: number;
  title: string;
  description: string;
  category: string;
  maxPlaces: number;
  availablePlaces: number;
  organizerId: string;
  status: string;
  schedules?: Schedule[];
}

File: ms-reservation/src/main/java/com/example/ReservationEvent/models/Event.java
package com.example.ReservationEvent.models;

import lombok.*;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Event {
    private Long id;
    private String title;
    private String description;
    private String category;
    private Integer maxPlaces;
    private Integer availablePlaces;
    private String organizerId;
    private String status;
    private List<Schedule> schedules; // Liste de simples objets Schedule
}